packages/dev-middleware/src/inspector-proxy/InspectorProxy.js

```diff
@@ -94,12 +94,13 @@ export default class InspectorProxy implements InspectorProxyQueries {
     response: ServerResponse,
     next: (?Error) => mixed,
   ) {
+    const path = url.parse(request.url).pathname;
```

Comment:
From CI, I think there's a formatting fix to apply in this file. Please run `yarn format` from the repo root.

nit: While we're here, we can modernise this call to use the WHATWG URL API.

```suggestion
    const path = new URL(request.url).pathname;
```

Reply:
Let's use `pathname` as well for clarity.

Reply:
I have made these changes in the latest commit - please see comment above usage of the WHATWG constructor and confirm that the change is appropriate.

Reply:
I might change this back before merging tbh - your comment raises good points (`url` is in many ways a more ergonomic API than `URL`), but you can leave this with us.